hydra_agent/agent.py

```diff
@@ -92,7 +94,14 @@ def get(self, url: str = None, resource_type: str = None,
             url = self.entrypoint_url + "/" + resource_type + "Collection"
             response = super().get(url, params=filters)
         else:
-            response = super().get(url)
+            if not bool(filters):
+                response = super().get(url)
+            else:
+                response_body = super().get(url)
+                # filtes can be simple dict or a json-ld
+                templated_url = 'http://localhost:8080' + \
```

Comment:
Priyanshu why are we hardcoding this URL?

Reply:
Earlier the idea was to construct URL from taking url from `@id` from the response. Since relative URLs are returned,  so the base_url was needed to be hardcoded. Now I have passed the whole URL at which request is made, and appended the query params to it. This approach helped remove the hardcoding of base url.